dev-haskell/cassava/cassava-0.5.2.0.ebuild

```diff
@@ -17,6 +17,8 @@ SLOT="0/${PV}"
 KEYWORDS="~amd64 ~x86"
 IUSE=""
 
+RESTRICT="test" # requires <quickcheck-2.14
```

Comment:
`RESTRICT=test` only stops tests from being ran. `USE=test` still tries to compile tests. You will need to remove `CABAL_FEATURES=test-suite` to avoid test build. It's also useful to add a note why it's required: does test fail to build if you lift upper bound on quickcheck?

Reply:
I can remove `test-suite`. I can also add a note.

Regarding lifting upper bound, i need to wait till aeson unmasked so i can unmask quickcheck too at the same time.